modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -168,7 +168,7 @@ private void _createFDSActionObjectDefintion(
 					ObjectFieldUtil.createObjectField(
 						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
 						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "method"), "method", false),
+						_language.get(locale, "variant"), "variant", false),
```

Comment:
Rename this field based on this [figma comment](https://www.figma.com/file/87FEbztFRRf0OqJJhiqT0U?node-id=228:68385&mode=design#572369876) . 
Find later that `async` and `headless` action types probably need this field

Reply:
`method` field is for `method` value of request in `async` action (`POST`, `PUT`, `DELETE` etc). Modal size would be a new field.